docs/release-guide.md

```diff
@@ -91,3 +130,23 @@ ensure it works on new language version. Often, only a small part of the
 new language version is placed behind a version pragma, with the rest of the
 changes being simply clarifications to the past spec. The upgrade guide should
 focus on the version-pragma-protected changes.
+
+## Release Steps
+
+Those steps are to be done when a new language revision is stabilized and time comes for a release (_remember that `.d`
```

Comment:
Maybe use something unrealistic like `x`.

Reply:
`.<rev>`? I usually use this form.